server/controllers/match/placementRequests/spaceBookingsController.ts

```diff
@@ -1,5 +1,5 @@
 import type { Request, RequestHandler, Response, TypedRequestHandler } from 'express'
-import type { ApType, Cas1NewSpaceBooking } from '@approved-premises/api'
+import type { ApType, Cas1NewSpaceBooking, PlacementCriteria } from '@approved-premises/api'
```

Comment:
Again, this should really reference `CasSpaceBookingCharacteristic` rather than `PlacementCriteria`. This might require `filterOutAPTypes` to have an update signature type, but it would make sense from here.

Reply:
True.  I added a fn to filter down from `PlacementCriteria` to `Cas1SpaceBookingCharacteristic` as the existing `filterOutApTypes` returns a  `Cas1SpaceCharacteristic` which contains more characteristics than we want.